src/SDAO.t.sol

```diff
@@ -475,6 +481,88 @@ contract SDAOTest is DssTest {
         vm.startPrank(sender);
         checkModifier(address(token), "SDAO/not-authorized", authedMethods);
     }
+
+    function testFile() public {
+        checkFileString(address(token), "SDAO", ["name"]);
```

Comment:
Could just combine these into `["name", "symbol"]` if support is added for the string[2] overload.

Reply:
Good point. Updated.